libraries/base/ETA/JString.hs

```diff
@@ -28,12 +28,17 @@ import GHC.Real
 import Data.String
 import Data.Char
 import Foreign
+import GHC.Show (Show(..))
+import GHC.Pack (unpackCString)
 
 -- TODO: Add rules to simplify
 -- fromString (unpackCString# "Hello world!"#) :: JString = JString "Hello world"#
 instance IsString JString where
   fromString = mkJString
 
+instance Show JString where
+    show = unpackCString
```

Comment:
For now, add quotes around the result and as we write more libraries with Java bindings, we'll get an understanding of whether it needs to be modified.